x-pack/plugins/security_solution/public/common/components/sourcerer/index.test.tsx

```diff
@@ -174,6 +174,29 @@ describe('Sourcerer component', () => {
     wrapper.find(`[data-test-subj="sourcerer-reset"]`).first().simulate('click');
     expect(wrapper.find(`[data-test-subj="config-option"]`).first().exists()).toBeFalsy();
   });
+  it('disables saving when no index patterns are selected', async () => {
```

Comment:
also think you can get rid of `async` on this line

Reply:
yep :)